### PR TITLE
doc/crimson: add SeaStore when using vstart.

### PR DESCRIPTION
**Pull request by Svelar**
doc/dev/crimson/crimson.rst: add SeaStore option when using crimson in vstart environment.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by Svelar** [comment]
@zdover23 Help review, please.

**Review by zdover23 — APPROVED** [review]

File: doc/dev/crimson/crimson.rst [hunk 1]
```diff
@@ -182,6 +182,9 @@ using ``vstart.sh``,
 ``--memstore``
     use the alienized MemStore as the object store backend.
 
+``--seastore``
+    use the SeaStore as the object store backend.
```
Comment (zdover23) [inline doc/dev/crimson/crimson.rst:186]: s/use the SeaStore/use SeaStore/

Remove "the". That is, if I understand correctly what Seastore is. The definite article should be removed from this sentence.

**Review by zdover23 — CHANGES_REQUESTED** [review]

File: doc/dev/crimson/crimson.rst (hunk 1 above)
Comment (Svelar) [reply doc/dev/crimson/crimson.rst]: Yes, I have modified, thanks.

**Review by zdover23 — APPROVED** [review]

**Comment by zdover23** [comment]
I'll backport this to Reef now (but only Reef now).

**Pull request merged**